### PR TITLE
Kvangreen

### PR DESCRIPTION
**Pull request by kvangreen**
- Created pages, moved a few. Created a controller but I think I did that wrong. 
- There's currently a weird error that doesn't make sense (syntax error on application.html.erb, line 37 (there are only 35 lines in the file): unexpected end_of_input: expected `end')

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -12,10 +12,24 @@
     <title>ValetBike</title>
   </head>
   <body>
+  <nav>
+  <div>
+    <p> <a href="app/views/pages/index.html">Home</a> <a href="app/views/pages/index.html">Pricing</a> <a href="app/views/pages/index.html">Map</a> <a href="app/views/pages/index.html">FAQ</a> <a href="app/views/pages/index.html">Pricing</a></p>
```
Comment (epartakki) [inline app/views/layouts/application.html.erb:17]: All of these would need their specific HTML links!

**Review by epartakki — COMMENTED** [review]
We have the idea! We just need to make sure that all the components are working with each other and that we can integrate these ideas to our bootstrap plan!

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -12,10 +12,24 @@
...
+  </div>
+  <div>
+    <% if THEY ARE LOGGED IN %>
```
[the remark this answers is not in the document]
Comment (epartakki) [reply app/views/layouts/application.html.erb:20]: What Kat has could work, it's just incorrect formatting

[the remark this answers is not in the document]
Comment (luciebrock) [reply app/views/layouts/application.html.erb:20]: do u want to try and reformat it today or do we want to drop it and just go with bootstrapping?

[the remark this answers is not in the document]
Comment (epartakki) [reply app/views/layouts/application.html.erb:20]: Up to you-- I do not intend to.

[the remark this answers is not in the document]
Comment (luciebrock) [reply app/views/layouts/application.html.erb:20]: does the existing html conflict with/get overwritten by bootstrap? I thought I remembered you mentioning that on thursday. if that's the case we should probably abandon it?

**Pull request closed without merging**